docs/core/docker/intro-net-docker.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Introduction to .NET and Docker
+description: Understanding Docker and .NET Core
+keywords: .NET, .NET Core, Docker
+author: jralexander
+ms.author: johalex
+ms.date: 10/11/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-docker
+ms.devlang: dotnet
+ms.assetid: 03c28597-7e73-46d6-a9c3-f9cb55642739
+---
+
+# Introduction to .NET and Docker
+
+ This article provides an introduction and conceptual background to working with .NET on Docker. 
+
+## Docker: Packaging your apps to deploy and run anywhere
+
+[Docker](https://docs.microsoft.com/en-us/dotnet/standard/microservices-architecture/container-docker-introduction/docker-defined) is an open platform that enables developers and administrators to build [images](https://docs.docker.com/glossary/?term=image), ship, and run distributed applications in a loosely isolated environment called a [container](https://www.docker.com/what-container). This approach enables efficient application lifecycle management between development, QA, and production environments.
+ 
+The [Docker platform](https://docs.docker.com/engine/docker-overview/#the-docker-platform) uses the [Docker Engine](https://docs.docker.com/engine/docker-overview/#docker-engine) to quickly build and package apps as [Docker images](https://docs.docker.com/glossary/?term=image) written in the [Dockerfile](https://docs.docker.com/glossary/?term=Dockerfile) [format that then is deployed and run in a [layered container](https://docs.docker.com/engine/userguide/storagedriver/imagesandcontainers/#container-and-layers).
+
+You can either create your own [layered images](https://docs.docker.com/engine/userguide/storagedriver/imagesandcontainers/#images-and-layers) as dockerfiles or use existing ones from a [registry](https://docs.docker.com/glossary/?term=registry), like [Docker Hub](https://docs.docker.com/glossary/?term=Docker%20Hub).
```

Comment:
I'd rephrase this: you don't create an image **as** a dockerfile, but the image is created **using** a dockerfile.

Reply:
Great point.